components/director/internal/domain/formationtemplate/fixtures_test.go

```diff
@@ -0,0 +1,98 @@
+package formationtemplate_test
+
+import (
+	"github.com/kyma-incubator/compass/components/director/internal/domain/formationtemplate"
+	"github.com/kyma-incubator/compass/components/director/internal/domain/formationtemplate/automock"
+	"github.com/kyma-incubator/compass/components/director/internal/model"
+	"github.com/kyma-incubator/compass/components/director/pkg/graphql"
+	"github.com/kyma-incubator/compass/components/director/pkg/pagination"
+)
+
+const (
+	testID                   = "d1fddec6-5456-4a1e-9ae0-74447f5d6ae9"
+	formationTemplateName    = "formation-template-name"
+	runtimeType              = "some-runtime-type"
+	runtimeTypeDisplayName   = "display-name-for-runtime"
+	artifactKindAsString     = "SUBSCRIPTION"
+	applicationTypesAsString = "[\"some-application-type\"]"
+)
+
+var (
+	applicationTypes = []string{"some-application-type"}
+)
```

Comment:
It's only one variable so we can remove the var block, even we can move it in the block below